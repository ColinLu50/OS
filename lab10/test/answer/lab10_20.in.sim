11615
370
752
376
728
364
