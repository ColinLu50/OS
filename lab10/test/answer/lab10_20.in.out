FCFS
374 -> 52 -> 279 -> 291 -> 322 -> 62 -> 17 -> 50 -> 356 -> 374 -> 73 -> 163 -> 313 -> 190 -> 283 -> 241 -> 159 -> 114 -> 25 -> 175 -> 342 -> 206 -> 305 -> 93 -> 243 -> 187 -> 322 -> 200 -> 267 -> 376 -> 151 -> 202 -> 317 -> 316 -> 13 -> 137 -> 35 -> 131 -> 191 -> 365 -> 145 -> 278 -> 62 -> 340 -> 84 -> 124 -> 371 -> 119 -> 278 -> 295 -> 68 -> 299 -> 12 -> 71 -> 367 -> 217 -> 212 -> 159 -> 177 -> 377 -> 140 -> 378 -> 191 -> 190 -> 202 -> 311 -> 253 -> 63 -> 261 -> 347 -> 219 -> 214 -> 66 -> 89 -> 139 -> 145 -> 217 -> 199 -> 75 -> 125 -> 270 -> 327 -> 22 -> 378 -> 99 -> 345 -> 325
11615
SSTF
374 -> 374 -> 376 -> 377 -> 378 -> 378 -> 371 -> 367 -> 365 -> 356 -> 347 -> 345 -> 342 -> 340 -> 327 -> 325 -> 322 -> 322 -> 317 -> 316 -> 313 -> 311 -> 305 -> 299 -> 295 -> 291 -> 283 -> 279 -> 278 -> 278 -> 270 -> 267 -> 261 -> 253 -> 243 -> 241 -> 219 -> 217 -> 217 -> 214 -> 212 -> 206 -> 202 -> 202 -> 200 -> 199 -> 191 -> 191 -> 190 -> 190 -> 187 -> 177 -> 175 -> 163 -> 159 -> 159 -> 151 -> 145 -> 145 -> 140 -> 139 -> 137 -> 131 -> 125 -> 124 -> 119 -> 114 -> 99 -> 93 -> 89 -> 84 -> 75 -> 73 -> 71 -> 68 -> 66 -> 63 -> 62 -> 62 -> 52 -> 50 -> 35 -> 25 -> 22 -> 17 -> 13 -> 12
370
SCAN
374 -> 371 -> 367 -> 365 -> 356 -> 347 -> 345 -> 342 -> 340 -> 327 -> 325 -> 322 -> 317 -> 316 -> 313 -> 311 -> 305 -> 299 -> 295 -> 291 -> 283 -> 279 -> 278 -> 270 -> 267 -> 261 -> 253 -> 243 -> 241 -> 219 -> 217 -> 214 -> 212 -> 206 -> 202 -> 200 -> 199 -> 191 -> 190 -> 187 -> 177 -> 175 -> 163 -> 159 -> 151 -> 145 -> 140 -> 139 -> 137 -> 131 -> 125 -> 124 -> 119 -> 114 -> 99 -> 93 -> 89 -> 84 -> 75 -> 73 -> 71 -> 68 -> 66 -> 63 -> 62 -> 52 -> 50 -> 35 -> 25 -> 22 -> 17 -> 13 -> 12 -> 0 -> 376 -> 377 -> 378
752
C-SCAN
374 -> 371 -> 367 -> 365 -> 356 -> 347 -> 345 -> 342 -> 340 -> 327 -> 325 -> 322 -> 317 -> 316 -> 313 -> 311 -> 305 -> 299 -> 295 -> 291 -> 283 -> 279 -> 278 -> 270 -> 267 -> 261 -> 253 -> 243 -> 241 -> 219 -> 217 -> 214 -> 212 -> 206 -> 202 -> 200 -> 199 -> 191 -> 190 -> 187 -> 177 -> 175 -> 163 -> 159 -> 151 -> 145 -> 140 -> 139 -> 137 -> 131 -> 125 -> 124 -> 119 -> 114 -> 99 -> 93 -> 89 -> 84 -> 75 -> 73 -> 71 -> 68 -> 66 -> 63 -> 62 -> 52 -> 50 -> 35 -> 25 -> 22 -> 17 -> 13 -> 12 -> 0 -> 378 -> 378 -> 377 -> 376
376
LOOK
374 -> 371 -> 367 -> 365 -> 356 -> 347 -> 345 -> 342 -> 340 -> 327 -> 325 -> 322 -> 317 -> 316 -> 313 -> 311 -> 305 -> 299 -> 295 -> 291 -> 283 -> 279 -> 278 -> 270 -> 267 -> 261 -> 253 -> 243 -> 241 -> 219 -> 217 -> 214 -> 212 -> 206 -> 202 -> 200 -> 199 -> 191 -> 190 -> 187 -> 177 -> 175 -> 163 -> 159 -> 151 -> 145 -> 140 -> 139 -> 137 -> 131 -> 125 -> 124 -> 119 -> 114 -> 99 -> 93 -> 89 -> 84 -> 75 -> 73 -> 71 -> 68 -> 66 -> 63 -> 62 -> 52 -> 50 -> 35 -> 25 -> 22 -> 17 -> 13 -> 12 -> 376 -> 377 -> 378
728
C-LOOK
374 -> 371 -> 367 -> 365 -> 356 -> 347 -> 345 -> 342 -> 340 -> 327 -> 325 -> 322 -> 317 -> 316 -> 313 -> 311 -> 305 -> 299 -> 295 -> 291 -> 283 -> 279 -> 278 -> 270 -> 267 -> 261 -> 253 -> 243 -> 241 -> 219 -> 217 -> 214 -> 212 -> 206 -> 202 -> 200 -> 199 -> 191 -> 190 -> 187 -> 177 -> 175 -> 163 -> 159 -> 151 -> 145 -> 140 -> 139 -> 137 -> 131 -> 125 -> 124 -> 119 -> 114 -> 99 -> 93 -> 89 -> 84 -> 75 -> 73 -> 71 -> 68 -> 66 -> 63 -> 62 -> 52 -> 50 -> 35 -> 25 -> 22 -> 17 -> 13 -> 12 -> 378 -> 377 -> 376
364
